venafi/resource_venafi_certificate_test.go

```diff
@@ -203,6 +203,37 @@ output "certificate" {
 output "private_key" {
 	value = "${venafi_certificate.token_certificate_custom_fields.private_key_pem}"
 }`
+	tokenValidDaysConfig = `
+%s
+resource "venafi_certificate" "token_certificate" {
+	provider = "venafi.token_tpp"
+	common_name = "%s"
+	san_dns = [
+		"%s"
+	]
+	san_ip = [
+		"%s"
+	]
+	san_email = [
+		"%s"
+	]
+	%s
+	key_password = "%s"
+	expiration_window = %d
+    issuer_hint = "%s"
+    valid_days =   %d
```

Comment:
nit... inconsistent indenting

Reply:
this seems to be an issue with the github code view, since on my IDE it look indented correctly.

Reply:
the issue is that those 2 lines use spaces while the rest of the file uses tabs for indentation

Reply:
oh, thank you @lhdomenech for the suggestion, now the issue is fixed